INSTALL.md

```diff
@@ -1,55 +1,36 @@
 
 ## Paths for future reference
-  * `/<ezpublish-community-root-dir>/`: The filesystem path where eZ Publish 5 is installed in
+  * `/<root-dir>/`: The filesystem path where eZ Platform is installed in
 
-    Examples: `/home/myuser/www/` or `/var/sites/ezpublish/`
-  * `/<ezpublish-community-root-dir>/ezpublish_legacy/`: **Legacy** aka **Legacy Stack** refers to the eZ Publish 4.x installation which is bundled with eZ Publish 5 inside `ezpublish_legacy/`
+    Examples: `/home/myuser/www/` or `/var/sites/<project-name>/`
 
-    Examples: `/home/myuser/www/ezpublish_legacy/` or `/var/sites/ezpublish/ezpublish_legacy/`
-  * `/<ezpublish-community-root-dir>/web/`: A directory meant to be the **DocumentRoot** of the eZ Publish 5.x installation (`ezpublish` & `ezpublish_legacy` are not supposed to be _viewable_ from user perspective)
+  * `/<root-dir>/web/`: A directory meant to be the **DocumentRoot** of the eZ Platform installation (`<root-dir>` is not supposed to be _readable_ from web server perspective)
 
-    Examples: `/home/myuser/www/web/` or `/var/sites/ezpublish/web/`
+    Examples: `/home/myuser/www/web/` or `/var/sites/<project-name>/web/`
 
-## Install all components
+## Clean install
 
-1. You can get eZ Publish using GIT with the following command:
+1. You can get eZ Platform using composer with the following commands:
 
        ```bash
-       git clone https://github.com/ezsystems/ezpublish-community.git
-       ```
-
-2. *Optional* Upgrade an eZ Publish installation
-
-       Follow normal eZ Publish upgrade procedures for upgrading the distribution files and moving over extensions as found for instance here:
-       http://doc.ez.no/eZ-Publish/Upgrading/Upgrading-to-5.0/Upgrading-from-4.7-to-5.0
-
-3. Install the dependencies with [Composer](http://getcomposer.org).
-
-       **Note: The following step will also install assets, if you prefer to install assets using hard copy or symlink
-               instead of default relative symlink, edit 'symfony-assets-install' setting in composer.json**
-
-       **Dev: For dev use change '--prefer-dist' for '--prefer-source' to get full git clones
-              and add '--dev' to get phpunit and behat installed.**
-
-       Download composer and install dependencies by running:
-       ```bash
-       cd /<ezpublish-community-root-dir>/
        curl -s http://getcomposer.org/installer | php
-       php -d memory_limit=-1 composer.phar install --prefer-dist
+       php -d memory_limit=-1 composer.phar create-project --prefer-dist ezsystems/ezpublish-community <project-name> <version>
+       cd /<project-name>/
        ```
+       
+      Options:
+      - `<version>`: `dev-master` to get current development version (pre release), `v2015.01` to pick a specific release, otherwise skip it to get latest stable release.
+      - For core development change '--prefer-dist' to '--prefer-source' to get full git clones, and add '--dev' to get phpunit and behat installed.
```

Comment:
`--dev` is the default behavior, so example for install should be more like:

```
php -d memory_limit=-1 composer.phar create-project ezsystems/ezpublish-community <project-name> <version> --prefer-dist --no-dev
```

And instruct here that one can remove `--no-dev` if they want dev/test tools.